addons/payment/static/src/js/payment_processing.js

```diff
@@ -54,6 +54,8 @@ odoo.define('payment.processing', function (require) {
                 }
                 else {
                     switch(data.error) {
+                    case "pg_error":
```

Comment:
`pg_error` could perhaps be named `tx_process_retry` or something that makes more obvious that it's ok-ish, and that we simply need to retry again?